testing/etckeeper/etckeeper.trigger

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# we are in trigger context so the apk database
+# contains info before the change takes effect
+/etc/etckeeper/pre-install.d/10packagelist
+
+LOCK="/lib/apk/db/lock"
+exec 200>"$LOCK"
+
+# wait for $LOCK before commit
+(flock --timeout 180 200 && etckeeper post-install) &
```

Comment:
yes ;-) was not happy with that trigger at all, but it was the only way to do it with current apk-tools. Hence this trigger (removal of it) is the motivation for the work in alpinelinux/apk-tools#2 i.